include/dca/phys/dca_step/cluster_solver/ctint/walker/tools/kernels_interface.hpp

```diff
@@ -31,6 +31,9 @@ void buildG0Matrix(linalg::MatrixView<Real, linalg::GPU> G0, const int n_init,
                    const bool right_section, DeviceConfiguration config,
                    DeviceInterpolationData<Real> g0_interp, cudaStream_t stream);
 
+template <typename Real>
+Real interpolateSlow(Real tau, int linindex, const DeviceInterpolationData<Real>& g0);
```

Comment:
Some documentation of why the name is Slow would be good here.

Reply:
done in 6c2ca2d784af39324b7098c8297178860697f355

Reply:
Thanks